setup.cfg

```diff
@@ -32,7 +32,7 @@ zip_safe = False
 # jwcrypto has a direct dependency on six, but does not list it yet in a release
 # Previously, cryptography also depended on six, so this was unnoticed
 install_requires =
-	django >= 2.2, <= 4.1
+	django >= 2.2, != 4.0.0
```

Comment:
restoring the block on 4.0.0 which had a bug, as noted here:

https://github.com/jazzband/django-oauth-toolkit/pull/1039#pullrequestreview-834284574